region/norfair/east/Cathedral.json

```
@@ -434,6 +444,11 @@
         "h_heatProof",
         {"refill": ["Energy", "Missile", "Super"]}
       ]
+    },
+    {
+      "link": [4, 4],
+      "name": "Crystal Flash",
+      "requires": ["h_canHeatedCrystalFlash"]
```

```suggestion
      "requires": ["h_canHeatedCrystalFlash"],
      "clearsObstacles": ["A"]
```
Looks like lower Geruta is the obstacle and upper Geruta is just avoided.  A PB at 4 would safely kill both and is a good position for preparing to collect the lava missile.

---

Ah, I was confused looking at node 4 of the other room, "Cathedral Entrance". Makes sense now